x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -16,7 +16,8 @@ import {
   injectedMetadataServiceMock,
   docLinksServiceMock,
 } from '@kbn/core/public/mocks';
-import { HttpService } from '@kbn/core/public/http';
+// FIXME plugins should not use core's internal service implementations
```

Comment:
so there's that

Reply:
Yea, this should use the `http` mocks instead:

https://github.com/elastic/kibana/blob/f609f092a5c179e2551cc68b97b72e97559a7b25/x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx#L30-L36

Reply:
I updated the test to make use of the `httpServiceMock` instead.
The tests pass, but I get a warning in console that @elastic/platform-deployment-management might want to take a look at:
```
console.error
    Warning: An update to AddPolicyToTemplateConfirmModal inside a test was not wrapped in act(...).
    
    When testing, code that causes React state updates should be wrapped into act(...):
    
    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */
    
    This ensures that you're testing the behavior the user would see in the browser. Learn more at https://fb.me/react-wrap-tests-with-act
        in AddPolicyToTemplateConfirmModal (created by ListActionHandler)
        in ListActionHandler (created by PolicyList)
        in PolicyList (created by TestComponent)
        in PolicyListContextProvider (created by TestComponent)
        in Provider (created by TestComponent)
        in TestComponent (created by WrapperComponent)
        in WrapperComponent
```